### DISCUSSION TITLE
How do you use Hires.fix with this fork?

### DISCUSSION THREAD

**Discussion by Michoko92** [vladmandic/sdnext#2001] [Q&A]
Hi, 

Sorry if this is obvious, but I spent quite some time figuring out how to use hires.fix with this fork, and I couldn't make it work.

I'm using a regular SD 1.5 model. I thought that in SDNext, the Hires.fix would be achieved with the second pass options? I first generate a 512x768 image, then I try to upscale it by a factor of 2. Even if I set a very low denoise, the final image I get is completely different (and actually set to 512x512 resolution). I really must do something wrong.

Is there a hires.fix option hidden somewhere, similar to A1111? Or do I have to use specific settings in the second pass UI? 

Thank you for your help!

**Comment by vladmandic** [comment]
your workflow is correct. i've just fixed wrong resizing, there was a bug with that.

**Reply by Michoko92** [reply]
Thank you for your responsiveness. However, even with doing a new Git Pull, I still get the same behavior (Version: 88fff06c Sun Aug 13 10:58:02 2023 +0000). Here are my settings:

![image](https://github.com/vladmandic/automatic/assets/3066173/361e3fb6-030a-46aa-8c7b-1a5814a31786)

I still get a final 512x512 image.

**Reply by vladmandic** [reply]
In your screenshot it says hires disabled, so that's normal. Hires is available for original backend, not diffusers (will be soon, but not yet)

**Reply by Michoko92** [reply]
Oh I see, I switched backends and now it works. Thank you so much for your quick answer. Cheers!
